sentry-rails/lib/sentry/rails/action_cable.rb

```diff
@@ -20,8 +20,9 @@ def capture(connection, transaction_name:, extra_context: nil, &block)
               scope.set_span(transaction) if transaction
 
               begin
-                block.call
+                result = block.call
                 finish_transaction(transaction, 200)
+                return result
```

Comment:
This `return` is not needed:

```suggestion
                result
```